tests/test_file_ops.py

```diff
@@ -0,0 +1,66 @@
+import shutil
+import os
+from pathlib import Path
+import sys
+sys.path.insert(1, "nesta_ds_utils/")
+import file_ops
+import pytest
+
+
+@pytest.fixture
+def test_output_path():
+    """generates pathlib.Path to dump intermediate test data
+
+    Yields:
+        pathlib.Path: output path to dump intermetiate test data
```

Comment:
Put type in function definition?
```suggestion
        Output path to dump intermetiate test data.
```